rpc/systema/agent2systema.proto

```diff
@@ -30,3 +33,7 @@ message ReviewInterceptResponse {
   // allowed to become active.
   string msg = 2;
 }
+
+message LeftIntercept {
```

Comment:
I wasn't sure what to name this... Not sure if `LeftInterceptInfo` would be better... naming is hard so open to suggestions.

Reply:
`Left` sounds weird.  Perhaps `RemovedIntercept`?  Or `InterceptRemoval`? I think I like `InterceptRemoval` best.